### DISCUSSION TITLE
Duvida sobre Atualização do GLPI.

### DISCUSSION THREAD

**Discussion by iramaceber** [LiveHelperChat/livehelperchat#2014] [Q&A]
Gostaria de saber se o Live Chat funciona para o GLPI 10.0.10. Pois fiz toda a instalação perfeitamente porém a o chat não aparece, nada acontece de diferente.

**Comment by remdex** [comment]
As I wrote in previous ticket. It's your installation issue. Live Helper Chat has nothing to do with your installation. Read article https://doc.livehelperchat.com/docs/debug how to debug and you migh see an error.
